Source/control.cpp

```diff
@@ -256,6 +256,16 @@ void SetButtonStateDown(int btnId)
 	panbtndown = true;
 }
 
+int CountNewlines(std::string_view str)

```

Comment:
No need for this function, you can simply do:

```c++
std::count(str.begin(), str.end(), '\n')
```

Reply:
Thanks, I did not know about that function.